admin/acceptance.py

```diff
@@ -345,25 +433,12 @@ def start_cluster(self, reactor):
             ])
             commands = commands.on(success=lambda _: zfs_commands)
 
-        certificates_path = FilePath(mkdtemp())
-        print("Generating certificates in: {}".format(certificates_path.path))
-        certificates = Certificates.generate(
-            certificates_path,
-            self.nodes[0].address,
-            len(self.nodes))
-
-        cluster = Cluster(
-            all_nodes=pvector(self.nodes),
-            control_node=self.nodes[0],
-            agent_nodes=pvector(self.nodes),
-            dataset_backend=DatasetBackend.zfs,
-            certificates_path=certificates_path,
-            certificates=certificates)
-
-        commands = commands.on(success=lambda _: configure_cluster(cluster))
-
         yield perform(make_dispatcher(reactor), commands)
 
+        cluster = yield configured_cluster_for_nodes(
+            self.nodes, self.dataset_backend_configuration
```

Comment:
`configured_cluster_for_nodes` takes a `dataset_backend` argument, which you're not passing in.